build.sbt

```diff
@@ -568,9 +567,9 @@ lazy val protobufSettings = Def.settings(
   libraryDependencies ++= {
     val scalapbVersion =
       if (isScala211.value) "0.9.8"
-      // freeze versions so that we don't depend on newer version, scalapb 0.11.13 depends on Scala 2.13.10 etc.
-      else if (scalaVersion.value == "2.13.11") "0.11.13"
-      else if (scalaVersion.value == "2.13.12" || scalaVersion.value == "2.13.13") "0.11.15"
+      // for SIP-51, freeze version to a ScalaPB built against the lowest Scala 2.13.x version we support
+      else if (EarliestScala213 == "2.13.11") "0.11.13"
```

Comment:
this effectively downgrades ScalaPB for 2.12.x and 2.13.12+ , but I believe it's OK as it already happened for some versions of 2.13 in https://github.com/scalameta/scalameta/pull/3722